src/Core/Suppressions/ISonarQubeIssuesProvider.cs

```diff
@@ -30,5 +30,10 @@ public interface ISonarQubeIssuesProvider : IDisposable
         /// Returns SonarQube suppressed issues for the specified project and file
         /// </summary>
         IEnumerable<SonarQubeIssue> GetSuppressedIssues(string projectGuid, string filePath);
+
+        /// <summary>
+        /// Returns all SonarQube suppressed issues
+        /// </summary>
+        IEnumerable<SonarQubeIssue> GetAllSuppressedIssues();
```

Comment:
This would be used in https://github.com/SonarSource/sonarlint-visualstudio/pull/2894